matter/si91x/siwx917/BRD2605A/autogen/sl_component_catalog.h

```diff
@@ -0,0 +1,54 @@
+#ifndef SL_COMPONENT_CATALOG_H
+#define SL_COMPONENT_CATALOG_H
+
+// APIs present in project
+#define SL_CATALOG_MATTER_ACCESS_CONTROL_PRESENT
+#define SL_CATALOG_MATTER_ADMINISTRATOR_COMMISSIONING_PRESENT
+#define SL_CATALOG_MATTER_BASIC_INFORMATION_PRESENT
+#define SL_CATALOG_MATTER_DESCRIPTOR_PRESENT
+#define SL_CATALOG_MATTER_DIAGNOSTIC_LOGS_PRESENT
+#define SL_CATALOG_MATTER_DOOR_LOCK_PRESENT
+#define SL_CATALOG_MATTER_ETHERNET_NETWORK_DIAGNOSTICS_PRESENT
+#define SL_CATALOG_MATTER_FIXED_LABEL_PRESENT
+#define SL_CATALOG_MATTER_GENERAL_COMMISSIONING_PRESENT
+#define SL_CATALOG_MATTER_GENERAL_DIAGNOSTICS_PRESENT
+#define SL_CATALOG_MATTER_GROUP_KEY_MGMT_PRESENT
+#define SL_CATALOG_MATTER_GROUPS_PRESENT
+#define SL_CATALOG_MATTER_ICD_MANAGEMENT_PRESENT
+#define SL_CATALOG_MATTER_IDENTIFY_PRESENT
+#define SL_CATALOG_MATTER_LOCALIZATION_CONFIGURATION_PRESENT
+#define SL_CATALOG_MATTER_NETWORK_COMMISSIONING_PRESENT
+#define SL_CATALOG_MATTER_OPERATIONAL_CREDENTIALS_PRESENT
+#define SL_CATALOG_MATTER_OTA_REQUESTOR_PRESENT
+#define SL_CATALOG_MATTER_POWER_SOURCE_PRESENT
+#define SL_CATALOG_MATTER_POWER_SOURCE_CONFIGURATION_PRESENT
+#define SL_CATALOG_MATTER_SOFTWARE_DIAGNOSTICS_PRESENT
+#define SL_CATALOG_MATTER_SWITCH_PRESENT
+#define SL_CATALOG_MATTER_THREAD_NETWORK_DIAGNOSTICS_PRESENT
+#define SL_CATALOG_MATTER_TIME_FORMAT_LOCALIZATION_PRESENT
+#define SL_CATALOG_MATTER_USER_LABEL_PRESENT
+#define SL_CATALOG_MATTER_BLE_PRESENT
+#define SL_CATALOG_MATTER_WIFI_NETWORK_DIAGNOSTICS_PRESENT
+#define SL_CATALOG_BTN0_PRESENT
+#define SL_CATALOG_SIMPLE_BUTTON_PRESENT
+#define SL_CATALOG_SIMPLE_BUTTON_BTN0_PRESENT
+#define SL_CATALOG_BTN1_PRESENT
+#define SL_CATALOG_SIMPLE_BUTTON_BTN1_PRESENT
+#define SL_CATALOG_LEDB_PRESENT
+#define SL_CATALOG_SIMPLE_LED_PRESENT
+#define SL_CATALOG_SIMPLE_LED_LEDB_PRESENT
+#define SL_CATALOG_ULP_TIMER_PRESENT
+#define SL_CATALOG_ULP_TIMER_TIMER0_PRESENT
+#define SL_CATALOG_CMSIS_OS_COMMON_PRESENT
+#define SL_CATALOG_DEVICE_INIT_NVIC_PRESENT
+#define SL_CATALOG_EMLIB_CORE_PRESENT
+#define SL_CATALOG_EMLIB_CORE_DEBUG_CONFIG_PRESENT
+#define SL_CATALOG_FREERTOS_KERNEL_PRESENT
+#define SL_CATALOG_KERNEL_PRESENT
+#define SL_CATALOG_IOSTREAM_PRESENT
+#define SL_CATALOG_MEMORY_MANAGER_PRESENT
+#define SL_CATALOG_NVM3_PRESENT
+#define SL_CATALOG_SEGGER_RTT_PRESENT
+#define SL_CATALOG_SLEEPTIMER_PRESENT
```

Comment:
should this be gated by icd/display since you are gating the init of sleeptimer by those?

Reply:
done @jmartinez-silabs